security-privacy-self-assessment.md

```diff
@@ -0,0 +1,77 @@
+# Security and Privacy Questionnaire
+
+[Security and Privacy questionnaire](https://www.w3.org/TR/security-privacy-questionnaire/)
+responses for the Device Posture API
+
+### 2.1. What information might this feature expose to Web sites or other parties, and for what purposes is that exposure necessary?
+
+This API exposes the posture of the device, a physical state of a device. The posture typically apply only to device with a foldable screen or dual screens devices (devices with two screens connected with a physical hinge). The posture is typically derived from the hinge angle sensor but also few other sensors (for example if the keyboard is detached or not).
+
+At the moment the specification exposes two postures :
+- folded
+- continuous
+
+In order for developers to take advantage of the new categories they need to know how the devices are being used so they can implement better responsive design in their application. Here are few simple use cases :
+
+- Avoiding content in the fold area (like buttons).
+- Providing a split user interface to leverage the additional screen estate.
+
+### 2.2. Is this specification exposing the minimum amount of information necessary to power the feature?
+
+The API design only exposes a defined set of postures which is ultimately determined by the OS/platform. For example the API doesn't expose the hinge value of the device.

```

Comment:
misses some commas, like "For example, "

It does expose the hinge value, but as two values :) doesn't expose the raw sensor values for the hinge angle